tests/monster_test_generated.rs

```diff
@@ -192,6 +222,7 @@ pub mod example {
 mod bitflags_color {
   flatbuffers::bitflags::bitflags! {
     /// Composite components of Monster color.
+    #[derive(Default)]
```

Comment:
It seems to me that deriving Default here will lead to invalid values. Could you point me to an explanation of how this works?

Reply:
The default will be 0,  no flags set, which I think is always valid for bit_flags enums